ci/ubuntu-16.04.docker

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu:16.04
+
+RUN apt-get update
+RUN apt-get install -y git cmake make g++ libclang-3.5-dev liblldb-3.5-dev clang-format-3.5 pkg-config libboost-filesystem-dev libboost-regex-dev libgtksourceviewmm-3.0-dev aspell-en libaspell-dev
```

Comment:
When doing two run statements with apt-get we risk cache invalidation. The one I posted makes sure that doesn't happen.